### DISCUSSION TITLE
Selecting specific range gates corresponding to only the cloud region

### DISCUSSION THREAD

**Discussion by MohitDahliya** [ARM-DOE/pyart#1619] [Q&A] [answered]
* Py-ART version: 1.18.3
* Python version: 3.11.5
* Operating System: Windows

### Description

I am trying to construct a Z-ZDR scatterplot at 21 degree elevation angle. Out of the total range gates available at this elevation, I need to select only those range gates, which correspond to the cloud region in order to remove noise or irrelevant data. How can I select only those gates which corespond to the cloud?

### What I Did

Haven't made any code yet. Can only do so after I get some headstart on which direction to proceed along.

**Comment by zssherman** [comment]
@MohitDahliya I'm going to move this to a discussion.

**Comment by zssherman** [comment, marked as answer]
You could always try the cloud mask found in:
https://arm-development.github.io/RadTraQ/build/html/source/auto_examples/plot_cloud_mask.html#sphx-glr-source-auto-examples-plot-cloud-mask-py

Not too sure if that is entirely what your looking for. For that angle you can extract sweeps for that specific sweep and then use a gatefilter to filter out the range gates you don't want.
https://github.com/ARM-DOE/pyart/blob/main/pyart/filters/gatefilter.

**Comment by MohitDahliya** [comment]
Thank you ! I applied your methodology and got some results. I am not sure how accurate these results are, but I am going to insert them into my manuscript. If face any issues in future, will ask again.
